PoetAssistant/src/AppDelegate.swift

```diff
@@ -30,7 +30,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		if CommandLine.arguments.contains("UITesting") {
 			Settings.clear()
 			Suggestion.clearHistory(completion: nil)
-			Poem(withText:"").saveDraft()
+			let doc = PoemDocument.loadSavedPoem()
```

Comment:
More cleanup for documents was put into `Settings`. Maybe it should go somewhere else